src/views/ward-leader.vue

```diff
@@ -293,6 +293,69 @@ export default {
     ]),
     sampleBallotFormPrefilled () {
       return `${SAMPLE_BALLOT_FORM}?ward=${this.ward}&party=${this.party}`
+    },
+    allCommitteePersons: function () {
+      let commPersons = this.committeePersons
+      // Get all divisions from ward boundaries
+      let allDivisions = this.wardBoundaries.features.map(x => x.properties.division)
+      // Default committe person data for Vacant divisions
+      let personDefault = function (ward, party) {
```

Comment:
Also, if you make it an arrow function (and maybe rename as `vacantPerson()`) you could use `this.ward` and `this.party` instead of taking them as params and take in the division and subdivision as arguments so you could have code below more like:
```
committeePersonsList.push(personData || vacantPerson(subDivisionId, division, subDivision));
```